### DISCUSSION TITLE
2.0-alpha1: save default or custom config?

### DISCUSSION THREAD

**Discussion by sebastian-luna-valero** [tdviet/fedcloudclient#189] [Q&A] [answered]
Hi,

I am testing `2.0-alpha1`. When I run:

```bash
fedcloud config create
```

It saves the [default settings](https://github.com/tdviet/fedcloudclient/blob/2.0-alpha1/fedcloudclient/conf.py#L128).

I was wondering whether the user would expect the current values to be saved instead? For example, the default value for `oidc_agent_account` is `egi` but I am using `myaccount` instead. Should `egi` or `myaccount` be saved with the `config create` command?

**Comment by tdviet** [comment]
You are right, it make sense to save the actual current values, not the default values. The `create` function was the first implemented one and at the moment, no other values than the default :-)

**Comment by tdviet** [comment, marked as answer]
Fixed :-) https://github.com/tdviet/fedcloudclient/commit/6e2c206ef1d2e65fd4cfc0a89136533b5b235c1c

**Reply by sebastian-luna-valero** [reply]
Excellent, thank you very much, it works!
